options.go

```diff
@@ -177,8 +197,38 @@ func IdentityResolverFunc(resolver IdentityResolver) Option {
 	}
 }
 
+// RingChecksumStatPeriod configures the period between emissions of the stat
+// 'ring.checksum-periodic'. Using a value <=0 (or RingChecksumStatPeriodNever)
+// will disable emission of this stat. Using a value in (0, 10ms) will return
+// an error, as that value is unrealistically small. Normal values must
+// therefore be >=10ms. RingChecksumStatPeriodDefault defines the default.
+func RingChecksumStatPeriod(period time.Duration) Option {
+	return func(r *Ringpop) error {
+		if period <= 0 {
+			period = RingChecksumStatPeriodNever
+		} else if period < 10*time.Millisecond {
+			return errors.New("RingChecksumStatPeriod invalid below 10 ms")
+		}
+		r.config.RingChecksumStatPeriod = period
+		return nil
+	}
+}
+
 // Default options
 
+// RingChecksumStatPeriodNever defines a "period" which disables
+// ring.checksum-periodic stat emission.
+const RingChecksumStatPeriodNever = time.Duration(-1)
+
+// RingChecksumStatPeriodDefault defines the default emission period for the
+// ring.checksum-periodic stat.
+const RingChecksumStatPeriodDefault = time.Duration(5 * time.Second)
```

Comment:
we have no current "defaults.go" or "config.go" which specifies these sorts of things -- they are scattered about. Both are reasonable IMO, and since there is no clear precedent, I will leave it where it is. If we start having more of these, then it is trivial to move it to a better location.